src/include/detail/ivf/index.h

```diff
@@ -168,14 +175,15 @@ int ivf_index(
     external_ids = read_vector<ids_type>(ctx, external_ids_uri, start_pos, end_pos);
   }
   return ivf_index<T, ids_type, centroids_type>(
-      ctx, db, external_ids, centroids_uri, parts_uri, index_uri, id_uri, start_pos, end_pos, nthreads);
+      ctx, db, external_ids, deleted_ids, centroids_uri, parts_uri, index_uri, id_uri, start_pos, end_pos, nthreads);
 }
 
 template <typename T, class ids_type, class centroids_type>
```

Comment:
We will need to refactor this into functions that deal with updates and those that don't.  We should also discuss index.h in general -- I didn't actually know it was being used.

Reply:
Avoided adding complexity to ingestions that don't need deleted_ids by only checking when `deleted_ids` is not empty